src/typings/react-ace.d.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, ReactElement } from "react";
+
+export default class Ace extends Component {
+  props: {
```

Comment:
After fiddling around with some types: I like the `Component<Props>` approach way more 👍